CONTRIBUTING.md

```diff
@@ -281,25 +281,20 @@ It is useful to list hosts that do not work in the comments of a `ruleset`.  Thi
 For easy reading, please avoid using UTF characters unless in the rare instances that they are part of the hostname itself.
 
 Example:
-
 ```xml
 <!--
-				Invalid certificate:
-								8marta.glavbukh.ru
-								forum2.glavbukh.ru (incomplete certificate chain)
-
-				Redirect to HTTP:
-								8marta2013.glavbukh.ru
-								den.glavbukh.ru
-
-				Refused:
-								e.glavbukh.ru
-								www.e.glavbukh.ru
-
-				Time out:
-								psd.glavbukh.ru
-								str.glavbukh.ru
-
+	Invalid certificate:
+		8marta.glavbukh.ru
+		forum2.glavbukh.ru (incomplete certificate chain)
+	Redirect to HTTP:
+		8marta2013.glavbukh.ru
+		den.glavbukh.ru
+	Refused:
+		e.glavbukh.ru
+		www.e.glavbukh.ru
+	Time out:
+		psd.glavbukh.ru
+		str.glavbukh.ru
```

Comment:
Should we use an [example domain](https://www.iana.org/domains/reserved) or a domain we control for this?

Reply:
We can, I can adjust to reflect that or you can.

Reply:
#19899 feel free to adjust/copy my changes.